observable.js

```diff
@@ -204,7 +204,7 @@ const [Observable, Subscriber] = (() => {
 
       if (!privateState.has(this)) {
         const controller = new AbortController();
-        controller.abort();
+        controller.abort(); // TODO we need the reason here - how to get it without subscriptionController?
```

Comment:
Without `privateState` this feels like an illegal invocation. Perhaps we should treat it as such? Is this behaviour specified?